apps/codeforafrica/src/lib/api.ghost/posts.js

```diff
@@ -0,0 +1,97 @@
+import fs from "fs";
+import path from "path";
+
+import GhostContentAPI from "@tryghost/content-api";
+
+import convertToCamelCase from "@/codeforafrica/utils/camelcaseKeys";
+
+const cacheDir = path.join(process.env.PWD, "public/data");
+
+function ghostAPI() {
+  return new GhostContentAPI({
+    url: process.env.GHOST_URL,
+    key: process.env.GHOST_API_KEY,
+    version: process.env.GHOST_API_VERSION || "v5.0",
+  });
+}
+
+function transformPost(post) {
+  const {
+    customExcerpt,
+    slug,
+    primaryTag,
+    publishedAt: publishedAtRaw,
+    excerpt: originalExcerpt,
+    ...other
+  } = convertToCamelCase(post);
```

Comment:
```suggestion
const {
    customExcerpt,
    excerpt: originalExcerpt,
    primaryTag,
    publishedAt: publishedAtRaw,
    slug,
    tags: originalTags,
    ...other
  } = camelcaseKeys(post, { deep: true });
```
1. Prefer listing props alphabetically,
2. It's better to also process all tags here once and for all.